lib/ui/views/swap/swap_view.dart

```diff
@@ -11,164 +20,352 @@ class SwapCryptocurrencyView extends StatefulWidget {
 }
 
 class _SwapCryptocurrencyViewState extends State<SwapCryptocurrencyView> {
-  String dropdownValue = 'BTC';
-  String dropdownValue2 = 'ETH';
+  final _formKey = GlobalKey<FormState>();
+  double? tokenAmount1 = 0;
+  double? tokenAmount2 = 0;
+
+  String dropdownValue = 'اختر';
+  String dropdownValue2 = 'اختر';
+
+  String balance1 = '';
+  String balance2 = '';
+
+  final _fromAmountController = TextEditingController();
+  final _toAmountController = TextEditingController();
+
   var items = [
-    'BTC',
+    'اختر',
     'ETH',
     'USDT',
-    'BNB',
-    'USDC',
+    'UNI',
+    'BAT',
   ];
-
+  double tokenExchangeRate1 = 2;
```

Comment:
What 2, 0.5 means as default value?

Reply:
it was a placeholder to make sure the variables are on the right place and the math is correct